src/test/java/com/training/sample/tests/Assignment5Test.java

```diff
@@ -0,0 +1,20 @@
+package com.training.sample.tests;
+
+import com.training.sample.pages.Assignment5;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Assignment5Test {
+    Assignment5 assignment5 = new Assignment5();
+
+    @Test
+    public void testGetStringIndex() throws Exception {
```

Comment:
The Exception is class inside this class there are many classes that are both checked and unchecked, so throws Exception is not good practice
Instead of throws Exception you can write throws RuntimeException
change for remaining also please